app/javascript/dashboard/i18n/dateFormat.js

```diff
@@ -0,0 +1,200 @@
+const dateFormatsByLocale = {
```

Comment:
Please take a look at `getIntlDateFormatForLocale`, it's a neat hack to use `Intl.DateTimeFormat` to get the date formats, you can use it to get the `fullDateTime` and `timeOnly` as well.

https://github.com/chatwoot/chatwoot/blob/2d35fa135b17fc68d594d2b7f0e693ec51fef924/app/javascript/dashboard/components/ui/DatePicker/helpers/DatePickerHelper.js#L70-L82

Adding a static list of these locales is not a great idea, very difficult to be exhaustive and also error prone